sdk/modelsrepository/Azure.Iot.ModelsRepository/src/RepositoryHandler.cs

```diff
@@ -156,8 +145,12 @@ private static Queue<string> PrepareWork(IEnumerable<string> dtmis)
                 if (!DtmiConventions.IsDtmi(dtmi))
                 {
                     ModelsRepoEventSource.Instance.InvalidDtmiInput(dtmi);
-                    string invalidArgMsg = string.Format(CultureInfo.CurrentCulture, ServiceStrings.InvalidDtmiFormat, dtmi);
-                    throw new ResolverException(dtmi, invalidArgMsg, new ArgumentException(invalidArgMsg));
+
+                    string invalidArgMsg =
+                        $"{string.Format(CultureInfo.CurrentCulture, ServiceStrings.GenericResolverError, dtmi)} " +
```

Comment:
I think at this point, you might as well not use string interpolation. We're already concatenating the second string for line break reasons. The only thing we're adding in is the space. May as well just concat all 3.